### PR TITLE
Add spark to list of potential index types in console

### PR DESCRIPTION
**Pull request by drcrallen**

### CODE REVIEW

File: src/client/factories/iUtils.coffee [hunk 1]
```diff
@@ -2,7 +2,7 @@ moment = require '../../../bower_components/moment/min/moment.min.js'
 
 module.exports = ->
   parseTaskId: (taskId) ->
-    m = taskId.match /^((hadoop_convert_segment)|index_(hadoop|realtime)|(archive))_(.+)_(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{3}Z)/
+    m = taskId.match /^((hadoop_convert_segment)|index_(hadoop|realtime|spark)|(archive))_(.+)_(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{3}Z)/
     type = m[3]
     type ||= m[2]
     throw Error("Can't parse #{taskId}") unless m
```
Comment (xvrl) [inline src/client/factories/iUtils.coffee]: we should avoid throwing an error here, so the console doesn't break with unknown task types

Comment (drcrallen) [inline src/client/factories/iUtils.coffee]: yes, but that is a feature above and beyond this PR

Comment (xvrl) [inline src/client/factories/iUtils.coffee]: can we file an issue then :)

Comment (drcrallen) [inline src/client/factories/iUtils.coffee]: Filed

**Pull request merged**